packages/contracts/contracts/plugin/PluginSetupProcessor.sol

```diff
@@ -105,16 +106,17 @@ contract PluginSetupProcessor is DaoAuthorizable {
     }
 
     /// @notice The struct containing the parameters for the `prepareUninstallation` function.
-    /// @param pluginSetupRef The PluginSetupRepo address + the current version of the plugin at which time it's getting uninstalled.
-    /// @param setupPayload see IPluginSetup.SetupPayload
+    /// @param pluginSetupRef The reference to the plugin setup to be used for the uninstallation.
```

Comment:
Generally, this doesn't say much. It should say that `pluginSetupRef` must be passed as the plugin's current version.. The same goes for other pluginSetupRef above.